### PR TITLE
add README

### PR DESCRIPTION
**Pull request by DPotapov93**

### CODE REVIEW

File: src/main/resources/db/changelog/changes/07-insert-users-table.yaml
Comment (d1sam) [inline src/main/resources/db/changelog/changes/07-insert-users-table.yaml:1]: this change is not related to readme.md

File: src/main/java/mate/academy/onlinebookstore01/service/impl/OrderServiceImpl.java
Comment (d1sam) [inline src/main/java/mate/academy/onlinebookstore01/service/impl/OrderServiceImpl.java:1]: this change is not related to readme.md

**Review by d1sam — CHANGES_REQUESTED** [review]
Remove changes that not related to this hw.

**Review by vovasalyha — APPROVED** [review]
Approving, but please improve `How to run the project` section after you add Docker support.
